source/ref_installation.md

```diff
@@ -0,0 +1,415 @@
+
+Create namespace for Tekton:
+
+```bash
+$ kubectl create namespace tekton-pipelines
+```
+
+Install Tekton Pipelines with [Helm chart](https://github.com/odahu/odahu-helm/tree/master/odahu-flow-tekton) provided by ODAHU team:
+
+```bash
+$ helm install odahu/odahu-flow-tekton --name tekton --namespace tekton-pipelines
+```
+
+### 2.7. Install [fluentd](https://www.fluentd.org/) with set of cloud object storage plugins
+
+In order to save models training logs to object storage of used cloud provider, a container with `fluentd` is used, in which a set of [plugins](https://www.fluentd.org/plugins/all#input-output) for popular cloud providers' object storages (AWS S3, Google Storage, Azure Blob) is added.
+Installation is done using a [fluentd Helm chart](https://github.com/odahu/odahu-helm/tree/master/odahu-flow-fluentd) provided by ODAHU team.
+
+First, create an object storage bucket:
+```bash
+$ gsutil mb gs://odahu-flow-test-store/
+```
+
+Create namespace for Fluentd:
+```bash
+$ kubectl create namespace fluentd
+```
+
+Install Fluentd with specified values. Full list of values you can see in [TODO: Helm chart repository]().
+
+Example:
+```bash
+$  cat << EOF > fluentd_values.yaml
+fluentd:
+  resources:
+    requests:
+      cpu: 100m
+      memory: 256Mi
+    limits:
+      cpu: 200m
+      memory: 512Mi
+output:
+  target: gcs
+  gcs:
+    authorization: keyfile
+    bucket: odahu-flow-test-store
+    project: my-gcp-project-id-zzzzz
+    private_key_id: 0bacc0b0caa0a0aacabcacbab0a0b00ababacaab
+    private_key: -----BEGIN PRIVATE KEY-----\nprivate-key-here\n-----END PRIVATE KEY-----\n
+    client_email: service-account@my-gcp-project-id-zzzzz.iam.gserviceaccount.com
+    client_id: 000000000000000000000
+    auth_uri: https://accounts.google.com/o/oauth2/auth
+    token_uri: https://oauth2.googleapis.com/token
+    auth_provider_x509_cert_url: https://www.googleapis.com/oauth2/v1/certs
+    client_x509_cert_url: https://www.googleapis.com/robot/v1/metadata/x509/service-account%40my-gcp-project-id-zzzzz.iam.gserviceaccount.com
+EOF
+
+$ helm install odahu/odahu-flow-fluentd --name fluentd --namespace fluentd --values ./fluentd_values.yaml
+```
+
+### 2.8. Install [Prometheus operator](https://github.com/coreos/prometheus-operator) (optional)
+
+TBD.
+
+## 3. Install ODAHU
+
+### 3.1. Install ODAHU services
+
+Create namespace for core ODAHU service:
+```bash
+$ kubectl create namespace odahu-flow &&\
+  kubectl label namespace odahu-flow project=odahu-flow
+```
+
+Create namespace for ODAHU deployment and label it for Istio injection:
+```bash
+$ kubectl create namespace odahu-flow-deployment &&\
+  kubectl label namespace odahu-flow-deployment project=odahu-flow istio-injection=enabled
+```
+
+Prepare YAML config with values for `odahu-flow` Helm chart.
+
+Example:
+```bash
+$  cat << EOF > odahuflow_values.yaml
+odahuFlowVersion: 1.0.0
+logLevel: debug
+rbac: true
```

Comment:
Let's simplify the values file.

Reply:
Done in c35c683823bc4bedc40272cafa686519adcc5856.